repo_policy_compliance/blueprint.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Provides API blueprint for flask to run the policy checks."""
+
+import logging
+import os
+import secrets
+from typing import cast
+
+from flask import Blueprint, Response, request
+from flask_httpauth import HTTPTokenAuth
+
+from . import Result, all_
+
+repo_policy_compliance = Blueprint("repo_policy_compliance", __name__)
+auth = HTTPTokenAuth(scheme="Bearer")
+runner_tokens: set[str] = set()
```

Comment:
Flask could run in a multi-process environment, saving the token inside a global variable might not be accessible by another flask instance in another process.

Reply:
Let's discuss this on the other comment above